### ISSUE TITLE
Mrglion faith rulers titles cannot be revoked by Holy Light

### ISSUE THREAD

**Issue by mrrichmine** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2#703]
**Describe your suggestion in full detail below:**

Mrglion faith rulers titles cannot be revoked by Holy Light
Mrglion faith considered "Hostile" by Holy Light
You can revoke titles of "Evil" religion rulers (And only if you have Tier II - Limited Crown Authority)

This leads to stupid De jure war outcomes like Murlok chief becomes vassal of Stormwind king.

**Comment by zumbak04** [comment]
Don't get your suggestion. Do you want every faith to consider murlocs evil?

**Comment by mrrichmine** [comment]
I wanted just Holy Light.
Or, you can rework revokation of titles, so titles of rulers of Hostile faith can be revoked

**Comment by zumbak04** [comment]
> Or, you can rework revokation of titles, so titles of rulers of Hostile faith can be revoked

That's one of the main differences between Evil and Hostile.

> I wanted just Holy Light.

That's too picky if it's just the Holy Light.

**Comment by MemeWizard** [comment]
> That's too picky if it's just the Holy Light.

Yeah, dwarves and elves still could have murlocs as vassals because their faith is different from a holy light

> Do you want every faith to consider murlocs evil?

What about Light and Order religious family instead of every faith?

**Comment by Grekopithikos** [comment]
Why would they not vassalize murlocs?

**Comment by zumbak04** [comment]
> Why would they not vassalize murlocs?

Agree. I don't see it as a problem. I would rather see the Holy Light convert murlocs.

**Comment by Grekopithikos** [comment]
Maybe a mechanic where if an Alliance/Horde race vassalizes a non Alliance/Horde race that does not follow the same religion as them, then they are free to revoke their titles
